CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ to [Semantic Versioning](http://semver.org/).
 
 ## [unreleased] Unreleased
 
+### Added
+- The container now registers Service Providers using its own `Container::get()` method, instead of the `new` keyword. This allows Service Providers to utilize dependency injection. (thanks @defunctl).
```

Comment:
I appreciate that you took care of the business of thanking yourself, I'm serious: it shows attention.